submissions/Eugene-CG/OOP-exercise/index.js

```diff
@@ -0,0 +1,77 @@
+class Inhabitant {
+  constructor(name, saying, gender) {
+    this.name = name;
+    this.saying = saying;
+    this.gender = gender;
+  }
+  getStr() {
+    return `${this.name} ${this.saying} ${this.gender} `;
+  }
+}
+class Animal extends Inhabitant {
+  constructor(name, saying, gender) {
+    super(name, saying, gender);
+  }
+  getProperties() {
+    return super.getStr() + this.species + this.paws;
+  }
+}
+class Human extends Inhabitant {
+  constructor(name, saying, gender) {
+    super(name, saying, gender);
+    this.legs = 2;
+    this.hands = 2;
+    this.species = "Homo sapiens";
+  }
+  getProperties() {
+    return super.getStr() + this.species + this.legs + this.hands;
+  }
+}
+class Cat extends Animal {
+  constructor(name, saying, gender) {
+    saying = saying || "meow";
```

Comment:
Why the `|| "moew"`. You cannot skip `saying` in your arguments list, the only way you can proc this is by passing `""` as a saying. Maybe just make all cats say "Meow" and don't event require a `saying` for animals?